datasets/coco2014/metadata.json

```diff
@@ -0,0 +1,240 @@
+      "name": "val2014.zip",
+      "contentUrl": "http://images.cocodataset.org/zips/val2014.zip",
+      "contentSize": " B",
+      "sha256": "",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "@type": "sc:FileObject",
+      "name": "test2014.zip",
+      "contentUrl": "http://images.cocodataset.org/zips/test2014.zip",
+      "contentSize": " B",
+      "sha256": "",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "@type": "sc:FileSet",
+      "name": "image-files",
+      "containedIn": [
+        "#{train2014.zip}",
+        "#{val2014.zip}",
+        "#{test2014.zip}"
+      ],
+      "includes": "*.jpg",
+      "encodingFormat": "image/jpeg"
+    },
+    {
+      "@type": "sc:FileObject",
+      "name": "annotations_trainval2014.zip",
+      "contentUrl": "http://images.cocodataset.org/annotations/annotations_trainval2014.zip",
+      "contentSize": " B",
+      "sha256": "",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "@type": "sc:FileSet",
+      "name": "caption_annotations-files",
+      "containedIn": [
+        "#{annotations_trainval2014.zip}"
+      ],
+      "includes": "annotations/captions_(val|train)2014.json",
+      "encodingFormat": "application/json"
+    },
+    {
+      "@type": "sc:FileSet",
+      "name": "person_keypoints_annotations",
+      "containedIn": [
+        "#{annotations_trainval2014.zip}"
+      ],
+      "includes": "annotations/person_keypoints_(val|train)2014.json",
+      "encodingFormat": "application/json"
+    },
+    {
+      "@type": "sc:FileSet",
+      "name": "instancesperson_keypoints_annotations",
+      "containedIn": [
+        "#{annotations_trainval2014.zip}"
+      ],
+      "includes": "annotations/instances_(val|train)2014.json",
+      "encodingFormat": "application/json"
+    },
+    {
+      "@type": "sc:FileObject",
+      "name": "image_info_test2014.zip",
+      "contentUrl": "http://images.cocodataset.org/annotations/image_info_test2014.zip",
+      "contentSize": " B",
+      "sha256": "",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "@type": "sc:FileSet",
+      "name": "imageinfo",
+      "containedIn": [
+        "#{image_info_test2014.zip}"
+      ],
+      "includes": "annotations/image_info_test.json",
+      "encodingFormat": "application/json"
+    }
+  ],
+  "recordSet": [
+    {
+      "@type": "ml:RecordSet",
+      "name": "split_enums",
+      "description": "Maps split names to semantic values.",
+      "key": "#{name}",
+      "field": [
+        {
+          "name": "name",
+          "description": "One of: train, val, test.",
+          "@type": "ml:Field",
+          "dataType": "sc:Text"
+        },
+        {
+          "name": "url",
+          "description": "Corresponding mlcommons.org definition URL",
+          "@type": "ml:Field",
+          "dataType": [
+            "sc:Url",
+            "wd:3985153"
+          ]
+        }
+      ],
+      "data": [
```

Comment:
Arbitrary JSON is not supported in schema.org. Can you use an external CSV for now? Later, we'll support https://schema.org/PropertyValue